content/news/2016/09/2016-09-27-webinar-recap-snaps-and-stripes-sharing-public-service-stories-with-snapchat.md

```diff
@@ -8,6 +8,7 @@ authors:
 
 topics:
   - accessibility
+  - best-practices
```

Comment:
Can we delete `- best-practices` on this one?

Reply:
Sure

Reply:
Just meant it's not a best practice, so I don't think we should keep it on this one specifically.